src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestNG">
+
+    <test name="PlayASong suite" preserve-order="false">
+        <classes>
+            <class name="PlayASongTests">
+                <methods>
+                    <include name="validatePlayOfTheSong"/>
```

Comment:
To run the test, please add `<include name="playSong"/>`

![image](https://user-images.githubusercontent.com/104761482/198561642-d554af7c-aeae-4e06-924e-d2ccab44df36.png)